src/main.cpp

```diff
@@ -125,6 +125,18 @@ void handleInput (unsigned char key, int x, int y) {
       break;
     case 'h':
       displayUsage();
+      break;
+    case '1':
+      cursor->setShape(1);
```

Comment:
in your comment for get/set shape, you note that the cursor shape options are 0 and 1, but here it seems like they are 1 and 2?